### PR TITLE
Fix delegation info

### PR DESCRIPTION
**Pull request by filippoweb3**

### CODE REVIEW

File: docs/learn/learn-polkadot-opengov.md [hunk 1]
```diff
@@ -532,16 +536,13 @@ unlocking period countdown begins after the account undelegates. There can be di
...
 :::
 
-Occasional delegation and undelegation calls are fee-free: creating an incentive for token holders
-to use this feature and ensure that wallets can do it “by default” without any cost to end-users. It
-is worth noting that a user delegating their voting power does not imply that the delegate will have
-control over the funds of the delegating account: they can vote with a user's voting power: but they
-won't be able to transfer your balance, nominate a different set of validators or execute any call
-other than voting on the defined call/s by the user.
+It is worth noting that a user delegating their voting power does not imply that the delegate will
+have control over the funds of the delegating account. That account can vote with a user's voting
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov.md]:
```suggestion
have control over the funds of the delegating user's account. That delegate's account can vote with a user's voting
```

File: docs/learn/learn-polkadot-opengov.md [hunk 2]
```diff
@@ -532,16 +536,13 @@ unlocking period countdown begins after the account undelegates. There can be di
...
+power but won't be able to transfer balances, nominate a different set of validators, or execute any
+call other than voting on the defined call/s by the user.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov.md]: Did you mean OpenGov tracks here?

```suggestion
call other than voting on the tracks defined by the user.
```

File: docs/learn/learn-polkadot-opengov.md [hunk 3]
```diff
@@ -532,16 +536,13 @@ unlocking period countdown begins after the account undelegates. There can be di
...
 
-With the new delegation features, the goal is to ensure the required support for proposals to be
-enacted is reached while maintaining the anonymity of voters and keeping the overall design
-censorship-free.
+The goal of delegations is to ensure the required support for proposals to be enacted is reached
+while maintaining the anonymity of voters and keeping the overall design censorship-free.
```
Comment (profradha) [inline docs/learn/learn-polkadot-opengov.md]: How is the voter anonymity is achieved when the delegation info is available to be queried on-chain?

**Review by DrW3RK — COMMENTED** [review]
Looks good. Some minor comments to be addressed.

File: docs/learn/learn-polkadot-opengov.md (hunk 3 above)
Comment (filippoweb3) [reply docs/learn/learn-polkadot-opengov.md]: It's not xD, that sentence does not make sense as people can get an ID. And anyway is pseudo-anonymity. I will delete that part

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**